region/crateria/west.json

```diff
@@ -236,7 +236,35 @@
           "to": [
             {
               "id": 7,
-              "requires": null,
+              "requires": [
```

Comment:
Good catch. That's a 60 damage that could logically make or break a short hellrun early in an entrando seed.